tests/sentry/options/test_manager.py

```diff
@@ -203,23 +203,23 @@ def test_flag_prioritize_disk(self):
             assert self.manager.get("prioritize_disk") == "foo"
 
     def test_db_unavailable(self):
-        with patch.object(Option.objects, "get_queryset", side_effect=Exception()):
+        with patch.object(Option.objects, "get_queryset", side_effect=RuntimeError()):
             # we can't update options if the db is unavailable
-            with self.assertRaises(Exception):
+            with self.assertRaises(RuntimeError):
                 self.manager.set("foo", "bar")
 
         self.manager.set("foo", "bar")
         self.store.flush_local_cache()
 
-        with patch.object(Option.objects, "get_queryset", side_effect=Exception()):
+        with patch.object(Option.objects, "get_queryset", side_effect=RuntimeError()):
```

Comment:
This RuntimeError stuff is originally so that we don't assertRaises(Exception), but might as well convert these other places to encourage use.